docs/web-support/rails.md

```diff
@@ -19,3 +19,45 @@ window.ApexCharts = require("apexcharts") // expose to window
 import ApexCharts from 'apexcharts'
 window.ApexCharts = ApexCharts
 ```
+
+Or, if you use `importmaps', you can:
+
+Download to local vendor directory (optional but recommended)
+
+```bash
+$ wget -O vendor/javascript/apexcharts.esm.js https://ga.jspm.io/npm:apexcharts@3.53.0/dist/apexcharts.esm.js`
```

Comment:
Instead of `@<version>`, we should use `@latest` here because that version may obsolete in the future.